website_sale_firstname/README.rst

```diff
@@ -0,0 +1,21 @@
+.. image:: https://img.shields.io/badge/license-LGPL--3-blue.png
```

Comment:
this is a new module, pls use new readme structure https://github.com/OCA/maintainer-tools/tree/master/template/module/readme